x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/type/DataType.java

```diff
@@ -27,32 +27,9 @@
 import static java.util.stream.Collectors.toUnmodifiableMap;
 
 public enum DataType {
-    UNSUPPORTED("UNSUPPORTED", null, 0, false, false, false),
```

Comment:
I went to add three more parameters and couldn't read anything. So I shifted it to a sort of named parameters with defaults thing.